components/provisioner/e2e_test/main_test.go

```diff
@@ -0,0 +1,165 @@
+package main_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/machinebox/graphql"
+	"github.com/stretchr/testify/assert"
+)
+
+const tenant = "3e64ebae-38b5-46a0-b1ed-9ccee153a0ae"
+const waitDelay = 10 * time.Second
+const waitTimeout = 15 * time.Minute
+
+type ProvisionResp struct {
```

Comment:
Is it common to shorten type names from `*Response` to `*Resp` in golang?

Reply:
"response" is shortened to "resp" in Go `net/http` examples, although in variable name, not in type name.
```
resp, err := http.Get("http://example.com/")
...
resp, err := http.Post("http://example.com/upload", "image/jpeg", &buf)
...
resp, err := http.PostForm("http://example.com/form",
	url.Values{"key": {"Value"}, "id": {"123"}})
```